### PR TITLE
Update and rename SETI-CommunityTechTree.netkan to SETI-CTT.netkan

### PR DESCRIPTION
**Pull request by Y3mo**
Yeah, unfortunately I know about the consequences. But since another modder uses ckan "conflicts" definition to carry out some personal vendetta, I can not use the original identifier anymore.

Sad to see ckan misused like this to the detriment of the users.

### CODE REVIEW

**Comment by plague006** [comment]
The reason we (practically) never change identifier is because it completely strands existing users without an upgrade path. Would you mind clarifying what's going on with the metadata? And if you want to be discrete feel free to email me at gmail (this username).

**Comment by Dazpoet** [comment]
Appending some thoughts on this PR below;

Since this PR exists I will assume that any attempts at negotion between the parties have broken down? Otherways discussing the issue and coming to a mutually beneficial solution would seem to be the best course of action.

Changing the identifier won't do much seeing as @BobPalmer also use self-controlled .netkan files and can/will just update them to the new identifier should this PR go through and the conflict is wanted.

The conflict was added only for 1.1 versions of the USI mods and the changes where put into place at the same time as the new USI mods where released as can be seen [here](https://github.com/BobPalmer/CKAN/commit/150685a69d09e19d68db39a486a4d24b7794a058).

With selfhosted metadata like that for both `SETI-CommunityTechTree` and the `USI suite of mods` we acctually have direct access to the specific wants of the mod author and in this case it is clear that the conflict is wanted. If this is to the detriment of our userbase as @Y3mo claims then a solution must be found since we always act in the interest of our users as is stated e.g. [here](https://github.com/KSP-CKAN/CKAN/blob/master/policy/de-indexing.md)

With that said I think that changing identifiers, for any reason, is a generally bad move and doing so specifically to circumvent the direct wishes of a modder with selfhosted metadata is an even worse move.

**Comment by BobPalmer** [comment]
So a few things, since I just got ping'd.

The author in question has consistently broken my stuff and caused me support issues.  I do not consider my mod constellation compatible with SetiCTT, and the very fact that @Y3mo hijacked the CTT name (versus using his own) has caused confusion.  This seems to me precisely the case that incompatibility is for (i.e. breaking all of my science experiments, etc.).

The author has been consistently rude towards me since he started modding, and quite frankly, if someone is being rude and causing me support issues, I do reserve the right to mitigate that.  

Perhaps if said author can adjust their behavior (i.e. accusing me of a 'vendetta' when in fact I am mitigating real support issues and conflicts) we can negotiate this.  An apology for the incessant rudeness, clarification that SetiCTT is *not* CTT, and assurance that parts in my mods will not be balanced/relocated/adjusted/etc. would be a good start.  It may also go a long way if said author stops publicly bashing my work.  These are things that make discussion possible.

In other words, if you're invited to the party and decide to trash the place, rearrange the furniture, and complain about the host, don't be surprised if you are shown the door and not asked to come back.

**Comment by BobPalmer** [comment]
(Addendum) I did remove the check from CRP as that was in error, there's no conflict with CRP, just my part/gameplay mods since to my knowledge, CRP is not mucked with.

**Comment by Y3mo** [comment]
@Dazpoet : "The conflict was added only for 1.1 versions of the USI mods and the changes where put into place at the same time as the new USI mods where released as can be seen here."

No, the conflict was certainly not put in place at the same time as the new USI mods were released, as can be seen by the time stamps of the changes. The conflict for all those mods was put in place right after (same day) this post was published:
http://forum.kerbalspaceprogram.com/index.php?/topic/95645-11105-seti-unmanned-before-manned-patreon/&page=72#comment-2491962
Which was a response (after asking forum mods for help to resolve the situation) to this post:
forum.kerbalspaceprogram.com/index.php?/topic/95645-11105-seti-unmanned-before-manned-patreon/&do=findComment&comment=2485281
Which was a response to this post:
http://forum.kerbalspaceprogram.com/index.php?/topic/95645-11105-seti-unmanned-before-manned-patreon/&page=72#comment-2482951
Specifically this passage:
"What does KSP 1.1 mean for the mods within this thread?
As far as I can see, nothing at all. Since that dreaded connection feature did not make it for 1.1, there seems to be little or no direct effect. Though there will probably be some indirect effect when the other mods update, since that is an opportunity to make changes."

Furthermore, the majority of the "conflicting" mods are not even affected by the SETI-CommunityTechTree in any way. Thus the "conflict" for the majority of those mods was not added due to the interaction of them and the SETI-CommunityTechTree, but for other reasons, as can be seen from the time stamps of the forum posts and the netkan changes.

I would also like to point out that the forum posts in no way mention RoverDudes mods or RoverDudes position as a modder. They are solely concerned with RoverDudes work for squad and thus RoverDudes position as a squad staff member.

**Comment by BobPalmer** [comment]
And see notes above RE resolution.  

@Dazpoet - In this case, it is in the interest of users not to have confusion regarding mod operation, tech tree placement, part behavior, etc. - this is a defensive measure due to the number of support issues SETI has caused me in the past, and due to the very poor behavior of it's author, it has been proven time and time again that discourse is not productive.

Point of reference - I have many positive interactions and interoperability discussions with a host of mod authors - and when those discussions take place in an environment of mutual respect, these things get sorted.  @Y3mo needs to work on that a bit, since he's been consistently abusive and rude to me since him coming onto the KSP modding scene, and has ignored previous requests to not mess with my mods as they cause support issues.

Until such time as all adjustments for all of my mods are removed from SETI, I have to see this as a conflict as it confuses the user (I use CTT, a name hijacked by Seti-CTT, which causes confusion), as well as adjustments to mod behavior - all of which cause me support issues, and are by their very definition a conflict.

**Comment by Y3mo** [comment]
@BobPalmer : SETIctt started as an addon to the CommunityTechTree and required the CommunityTechTree to work. It thus followed the convention of addons to add to the base version of the name.

I m not sure what you mean with "consistent breaking" of your stuff, but I suspect you refer to the old 0.90 SETI-BalanceMod which is dead for about a year now and there are no plans to bring it back. This is the first time I hear about breaking science experiments by my tech tree mod. Any hint to the specifics would allow me to remove those issues.

About being rude to you, how is "dreading" a new stock feature because of the workload it brings being rude to you? I never mentioned you in this regard and I quite frankly did not even know you were the one responsible for that feature for quite some time after it was announced. I also acan not remember addressing you personally with regards to that feature or any other feature you implemented for squad.

Since I tried to stay out of your threads for over a year now, the main interaction between us was, when you came into the SETI thread, which was often coupled with snide remarks or flamebaiting in my direction. A rather one sided rudeness within the last year from my perspective, but of course that is subjective as well.

About the support issues, I can only assume that this again is mainly about the old SETI-BalanceMod which died with KSP 0.90. Please correct me if I m wrong about it.

About the SETIctt vs CTT: I m not aware of people confusing the two, but that may be because I m keeping out of your threads as you wished. I m taking you word for it and just added disclaimers to the thread download location, spacedock.info description and thus CKAN description, that SETIctt is not the CTT (should be updated on CKAN display when the bot next checks for it).

About the changing of mod parts:
Since the death of the old SETI-BalanceMod for KSP 0.90, I tried my best to avoid the issues you complained about over a year ago. I even use MM statements like this one for a long time now where possible:
:HAS[~author[*RoverDude*]]
To reiterate, the old SETI-BalanceMod is long dead, there are no plans to bring it back. And I planned to continue to avoid the issues you complained about.

To my knowledge, I never bashed your modding work, quite the opposite (though I concede that suggestions can be seen as critique, but I stopped suggesting stuff with regards to your mods a long time ago). In any case, I did not plan to start bashing your modding work.

With regards to the features announced for official squad updates:
As a customer, I reserve the right to critisize squad and possible changes to the game I paid for. Keeping the tone on the forums civil and sanctioning/warning about criticising squad is the job of the moderators. Vigilanty justice is imho not the way to go.
I do not remember bashing you personally (for the most time I did not even know that you were responsible for that specific feature) and I did not plan to start doing so.

**Comment by BobPalmer** [comment]
@Y3mo , then I'm sure you won't mind removing all of the USI references from this mod, yes?  If there is no change to any of my mods in anything by SETI, then I'd call that reasonable.

Also, there's a conflict in the way SETI changes reusable experiments that has proven to be very detrimental to my users in the past - I will need my parts opted out of that as well, as it causes both support requests and corresponding confusion for my users.

In the future, for mods like this, you would be better served to use an opt-in model instead of making assumptions that folks wish to participate.

**Comment by Dazpoet** [comment]
Clearing out my factual error first, I saw the commit with the conflicts being made at the same time as our metadata indexed the new versions. I seem to have forgotten that updated netkans overwrites the most recent metadata as if there was a new version. I'm sorry about that.

However I maintain that changing the identifier does not appear to be the solution to the situation we see unveiling here.

**Comment by Y3mo** [comment]
@BobPalmer 

With regards to your last post:
You know, mutual respect must come from both sides. I will not mix up official squad feature critique with modding work or personally critisizing you.
I tried to avoid conflict with you over the last year, even when you were flamebaiting time and again in the SETI thread. I did not resurrect the old SETI-BalanceMod and the USI configs within it. I even tried to help occasionally (eg pull request when you forgot that USI tools was a dependency for USI Life Support).

But honestly appeasement only goes so far.
CTT is not your mod. It is made for the community. I m part of the community. There are good reasons for the name. Stop trying to dictate what a community member can do with a community mod which you did not even create. A tech tree mod changes node positions and part positions. I can hardly push manned flight to a 45 science node but offer life support before that. That simply does not make any sense. I will leave your relative positions as good as I can, but I will not let you dictate the shape or gaemplay progression of my tech tree mod. If your parts break that progression, I try to mitigate that compatibility issue as good as possible, keeping your interests in mind as well.

The science definitions you refer to (mystery goo and materials bay) are not part of your mods, they are part of the stock game. As a customer I will not let you dictate how my mod changes the stock game. I will try to mitigate compatibility issues with your mod as good as possible, keeping your interest in mind as well.
However I made those changes opt-in. The opt-in is called installing my mod, which is called "rebalance". No one is forced to install it. I make those changes clear at the very top of my forum post, in red letters.

Appeasement only goes so far.
I tried for the last year. I omitted the configs you complained about a year ago. I even explicitly excluded your mods from general configs where possible. I offered support for your support-workload. When you declined, I kept out of your threads even if you did not keep out of mine. I tried to help on github when seeing issues with your mods independent of SETI. I toned down my critique of your squad work, after learning about your involvement, even though I m a paying customer and that affects me.

CTT is not your mod, SETIctt is not your tech tree. Stock science definitions are not your domain.

If you only leave me the choice of submission to your wishes whereever your opinion differs from mine (regardless whether your mods are involved or not)
or escalating conflict out of all proportion (by harassment and getting out of your way to force your opinion on me),
then I will stop with the appeasement and defend myself.

**Comment by BobPalmer** [comment]
Based on your last reply, it's apparent that there will be no resolution to this conflict.

From a policy standpoint, it seems pretty simple.  If Mod A is directly modifying Mod B against the express wishes of the creator of Mod B, and if the creator of Mod A refuses to remove this behavior, then mod B takes precedence, as Mod A is taking a direct action to the detriment of the users of Mod B.

And yes, you are free to do whatever you wish to do with your own mod.  But I am under no obligation to support it, and when it causes conflict, I am well within my rights to mitigate it.

**Comment by Y3mo** [comment]
@BobPalmer 

I added the disclaimer to SETIctt in every download location as you wished.
My feature related comment in the thread was neither rude nor aimed at you.
I tried to avoid any issues with you or your mods for the last year.

There were and are other tech tree mods moving your parts around, that is the way tech trees work. But you only take offense with SETIctt.

If you wish to dictate what people can mod, you should make your own game and disable modding. But instead you try to force your opinions on an existing community, to the detriment of the community members.

Most of the mods you added the "conflict" to are not even affected by SETIctt at all and never were. So the "conflict" was just introduced by your addition of the "conflict" within the netkan file. And that was a response to a simple comment towards a proposed squad feature you happen to work on...

You are really going to claim that this is not about your ego?

Well, if you consider it to be your right to artificially create conflict between your mods and other mods to the detriment of the users, then I consider it my right to resolve those artificial conflicts.

**Comment by Y3mo** [comment]
Thankfully CC 4.0 BY-NC-SA allows for easy resolution of compatibility issues since you are hellbent on conflict.

I really would have preferred a peaceful solution.

**Comment by BobPalmer** [comment]
And so does ARR.

**Comment by Dazpoet** [comment]
This thread have now left the subject of the PR and as such I'm locking it for now.

**Comment by Dazpoet** [comment]
Also closing this pullrequest by request of the author
